src/styles/Navigation.css

```diff
@@ -1,21 +1,44 @@
 .nav-bar {
-  height: 80px;
+  height: 100px;
   width: 100%;
   display: flex;
-  gap: 30px;
+  gap: 40px;
   align-items: center;
   padding: 0 50px;
-  margin-bottom: 20px;
+  margin-bottom: 50px;
+  background-color: #fff;
 }
 
 .nav-side {
   display: flex;
   justify-content: space-between;
-  gap: 620px;
+  gap: 580px;
+  width: 40%;
+}
```

Comment:
- [ ] Great job so far. Kindly consider aligning your nav items center as it's currently misaligned and due to that your design doesn't match that of the provided design in the project requirements. Kindly consider fixing that issue as it would help make your design look more professional. See below for more details:


<table>
<tr>
<th align=“middle”>orignal design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="863" alt="Screenshot 2023-08-24 at 17 23 12" src="https://github.com/Bhekisile/Bookstore/assets/107192076/7ac55d4a-60e4-4bdd-ba32-8a05686670a3">
</td>
<td>
<img width="1299" alt="Screenshot 2023-08-24 at 21 58 00" src="https://github.com/Bhekisile/Bookstore/assets/107192076/ab80c91e-0b61-497d-9975-0d4df576a6b5">
</td>
</tr>
</table>